frontend/src/metabase/parameters/utils/fields.js

```diff
@@ -6,12 +6,6 @@ export function hasFieldValues(parameter) {
   return parameter.fields.some(field => field.hasFieldValues());
 }
 
-export function getFieldIds(parameter) {
```

Comment:
After some recent refactors this function is no longer in use. I'm removing it here, now that the dashboard `UiParameters` don't have `field_ids` or `field_id` properties.